### PR TITLE
Search: fix front end query search condition

### PR DESCRIPTION
**Pull request by ArturWierzbicki**
follow up https://github.com/grafana/grafana/pull/55526

### CODE REVIEW

File: public/app/features/search/service/searcher.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export function getGrafanaSearcher(): GrafanaSearcher {
     const useBluge = config.featureToggles.panelTitleSearch;
     searcher = useBluge ? new BlugeSearcher(sqlSearcher) : sqlSearcher;
 
-    if (useBluge && location.search.indexOf('do-frontend-query')) {
+    if (useBluge && location.search.indexOf('do-frontend-query') !== -1) {
```
Comment (joshhunt) [inline public/app/features/search/service/searcher.ts:16]: Does `location.search.includes()` also work? That more clearly states your intention.

**Review by ashharrison90 — APPROVED** [review]
lgtm and fixes the bug, thanks @ArturWierzbicki 🙌

**Comment by ashharrison90** [comment]
as an aside, we should probably have a unit test for this. can be done in a follow-up PR though

File: public/app/features/search/service/searcher.ts (hunk 1 above)
Comment (ArturWierzbicki) [reply public/app/features/search/service/searcher.ts]: yep thats better!

**Pull request merged**